doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,386 @@
+
+      * Universal version of SEGGER J-Link.
+        This is required for SEGGER J-Link to work correctly with both Intel and ARM assemblies.
+
+.. _gs_installing_toolchain:
+.. _gs_installing_tools:
+
+.. rst-class:: numbered-step
+
+Install the |NCS| toolchain
+***************************
+
+The |NCS| :term:`toolchain` includes the Zephyr SDK and then adds on top of it tools and modules required to build |NCS| samples and applications.
+These include the :ref:`required tools <requirements_toolchain_tools>`, the Python dependencies, and the :ref:`GN tool <ug_matter_gs_tools_gn>` for creating :ref:`ug_matter` applications.
+
+Depending on your preferred development environment, install the |NCS| toolchain:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage toolchains`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install Toolchain`.
+         The list of available toolchain versions appears in the |VSC|'s quick pick.
+      #. Select the toolchain version to install.
+         The toolchain version should match the |NCS| version you are going to work with.
+         If you have received a custom URL for installing toolchain, you can provide it using the :guilabel:`Change Toolchain Index` button in the quick pick's header.
+         The toolchain installation starts in the background, as can be seen in the notification that appears.
+
+      When you install the toolchain for the first time, the installed version is automatically selected for your project.
+
+   .. group-tab:: Command line
+
+      1. Open a command prompt window.
+      #. Run the following command to install the nRF Util ``toolchain-manager`` command:
+
+         .. code-block:: console
+
+            nrfutil install toolchain-manager
+
+      #. Run the following command to list the available toolchain installations:
+
+         .. code-block:: console
+
+            nrfutil toolchain-manager search
+
+         The versions from this list will be *toolchain_version* in the following step.
+      #. Run the following command to install the toolchain version of your choice:
+
+         .. parsed-literal::
+            :class: highlight
+
+            nrfutil toolchain-manager install --ncs-version *toolchain_version*
+
+         For example:
+
+         .. code-block:: console
+
+            nrfutil toolchain-manager install --ncs-version v2.0.0
+
+         The toolchain version should match the |NCS| version you are going to work with.
+
+      The toolchain is installed by default at :file:`C:/ncs/toolchains` on Windows, :file:`~/ncs` on Linux, and :file:`/opt/nordic/ncs` on macOS.
```

Comment:
```suggestion
      The toolchain is installed by default at :file:`C:\ncs` on Windows, :file:`~/ncs` on Linux, and :file:`/opt/nordic/ncs` on macOS.
```
Alternatively add the "toolchains" directory to all the paths

Reply:
I'll keep forward slash, as it is also supported on Windows.

Reply:
On windows in command prompt? Are you sure?

Reply:
Yes